developer/src/ext/Makefile

```diff
@@ -1,53 +0,0 @@
-#
-# Keyman Developer ext Makefile - 3rd party components
-#
-
-NOTARGET_SIGNCODE=yes
-
-!ifdef NODELPHI
-TARGETS=.virtual
-!else
-TARGETS=browse4folder jedi mbcolor scfontcombobox common
-!endif
-
-CLEANS=
-
-!include ..\Header.mak
-
-# ----------------------------------------------------------------------
-
-browse4folder: .virtual
-    cd $(DEVELOPER_ROOT)\src\ext\browse4folder
-    $(MAKE) $(TARGET)
-
-jedi: .virtual
-    cd $(DEVELOPER_ROOT)\src\ext\jedi
-    $(MAKE) $(TARGET)
-
-mbcolor: .virtual
-    cd $(DEVELOPER_ROOT)\src\ext\mbcolor
-    $(MAKE) $(TARGET)
-
-scfontcombobox: .virtual
-    cd $(DEVELOPER_ROOT)\src\ext\scfontcombobox
-    $(MAKE) $(TARGET)
-
-# ----------------------------------------------------------------------
-
-common: .virtual
-    cd $(COMMON_ROOT)\ext
-    $(MAKE) $(TARGET)
-
-#
-#
-#
-
-reset:
-    $(DEVTOOLS) -rp
-    $(DEVTOOLS) -ri
```

Comment:
Huh... so this time, the `reset` target disappears entirely.  No corresponding `reset` action (or addition to `clean`, if that'd make sense.)

Reply:
Yeah see my commentary on the reset action. I should probably remove it altogether.